sources/karmashapes_remarks.md

```diff
@@ -0,0 +1,28 @@
+### Karmashapes
+
+Properties with the `ks` prefix are from Karmashapes.
+
+**Examples:** `ks:id` for Karmashapes identifier concordance and `ks:name` for place names. 
+
+Karmashapes is included in Who's On First's administrative data repo in India. Who's on First doesn't always use the Karmashapes geometry, so review that in `src:geom`. When we do, some Karmashapes features source from friendly CC0 or Public Domain licensed projects, and others from a stricter CC-BY project.
+
+Please note the following with regard to Quattroshapes features that require attribution:
+
+* National Geospatial Intelligence Agency (NGA)
+	* URL: https://geonames.nga.mil/geonames/GNSHome/welcome.html
+	* License: Public Domain
+* Socioeconomic Data and Applications Center (SECAC) / Center for International Earth Science Information Network (CIESIN);
+	* URL: https://sedac.ciesin.columbia.edu/data/set/india-india-village-level-geospatial-socio-econ-1991-2001/data-download
+	* License: CC-BY
+* Pradhan Mantri Gram Sadak Yojana (PMGSY);
+	* URL: https://geosadak-pmgsy.nic.in/opendata/
+	* License: CC-BY (equivalent)
+* Global Human Settlement Layer (GHSL);
+	* URL: https://ghsl.jrc.ec.europa.eu/ghs\_stat\_ucdb2015mt\_r2019a.php
+	* License: CC-BY
+* ESRI’s Sentinel-2 10m Land Use/Land Cover;
```

Comment:
I think this is actually `Impact Observatory by National Geographic Society, Microsoft, and Esri`

And `Sentinel-2 10m Land Use/Land Cover (January 2022)` is the data layer (note with new sub-bullet).

Reply:
And no `;` at end of the line